sportsref/ncaaf/players.py

```diff
@@ -174,3 +246,67 @@ def awards(self):
         for yr, award in zip(*[iter(results)]*2):
```

Comment:
haha don't write code like this, it's way too confusing. (I realize this was me. just noticed it and not sure what I was thinking lol). I must have really badly wanted to do this in one line, but that's not worth sacrificing readability